tools/appliance/systemvmtemplate/scripts/configure_grub.sh

```diff
@@ -20,6 +20,8 @@ set -e
 set -x
 
 function configure_grub() {
+  # Remove old kernel
+  apt-get remove -y --purge linux-image-amd64 linux-image-4* || true
```

Comment:
@weizhouapache would this help, essentially saying remove all the old 4.x kernels. As the new/LTS Linux kernel are in the 5.x series - https://www.kernel.org/category/releases.html